### PR TITLE
Add text-color to .navbar-dark

### PR DESCRIPTION
**Pull request by shirohana**
I add something useful in .navbar-dark.
It will override the old color.
result like follows:

### CODE REVIEW

File: scss/mixins/_text-emphasis.scss [hunk 1]
```diff
@@ -10,3 +10,14 @@
...
+@mixin text-emphasis-variant-inverse($parent, $color) {
+  #{$parent} {
+    color: $color;
+  }
+  a#{$parent} {
+    @include hover-focus {
+      color: lighten($color, 10%);
+    }
+  }
+}
```
Comment (Shirohana) [inline scss/mixins/_text-emphasis.scss]: oops.
wait a second :3

Comment (Shirohana) [inline scss/mixins/_text-emphasis.scss]: should I amend to orig. commit?

Comment (kkirsche) [inline scss/mixins/_text-emphasis.scss]: Yes.

**Comment by Shirohana** [comment]
Is it correctly?
Sorry about that took lots of time :(

**Comment by cvrebert** [comment]
Personally, I agree with the earlier criticisms that utility classes shouldn't be subject to mutation like this.
Adding a new set of "dark-friendly" text emphasis classes might be a better way to go.

**Comment by Shirohana** [comment]
Ok, let me do some change :)

**Comment by Shirohana** [comment]
I move them to _utilities.scss.
I'll squash commits if everything is fine.

**Comment by Shirohana** [comment]
Oops!
It will not work inside .navbar-dark (´・ω・｀)

Which way do you like?
!important? double the class? do again in .navbar-dark?
Or something suggest for me (´・д・｀)?

**Comment by mdo** [comment]
Sorry, for now, I don't think we need to duplicate these utilities. Thanks for raising the issue and providing code a few times! <3

**Pull request closed without merging**